resources/lang/en/admin/locations/table.php

```diff
@@ -20,4 +20,22 @@
     'parent'                    => 'Parent',
     'currency'                  => 'Location Currency',
     'ldap_ou'                   => 'LDAP Search OU',
-    ];
+    'company'                   => 'Company',
```

Comment:
I think some of these have already been moved into the `general.php` top level language file to avoid duplication, since we use "Company" (etc) in a lot of places, and it would be dumb (and was dumb for me to make it this way) to require the same strings be translated for each section.)

Reply:
Mhh... I searched through the file but was not able to find it... Might have overlooked it somewhere and I still need to sift through all my changes and review them because I believe that I have some duplications :D

Gonna sift through it starting around ... mhh... maybe Wednesday next week as my vacation is coming to an end tomorrow and I need to work through everything that has been sitting since my vacation that started at the start of December 😅 

You will probably see a lot of commits coming the next week!

Reply:
Yep, totally missed it. Fixed it in the next commit!